task-3_mongoDB-p1/README.md

```diff
@@ -0,0 +1,92 @@
+```
+mongoimport --db frontcamp --collection restaurants --file C:/data/db/restaurants.json
+```
+
+# task 3:
+
+1) RESULT: 728
```

Comment:
Please add query like as 
>db.restaurants.find...

Reply:
I use Compass for this task and in Compass you should create query without db.restaurants.find...
ps. query for Compass you can find at string 11

Reply:
Please attach a screenshot with result equals 728

Reply:
![image](https://user-images.githubusercontent.com/45118964/68540782-6a3d1b00-03a8-11ea-976b-91457e17a7aa.png)

Reply:
![image](https://user-images.githubusercontent.com/45118964/68540798-9062bb00-03a8-11ea-8ef4-5414bab40a5e.png)